application/lib/settings.dart

```diff
@@ -6,21 +6,21 @@ import 'disposition.dart';
 const double kSettingsWidth = 400.0;
 
 class SettingsTab extends StatefulWidget {
-  const SettingsTab({Key key}) : super(key: key);
+  const SettingsTab({Key? key}) : super(key: key);
```

Comment:
(no need to do it in this PR but fwiw, `({Key? key}) : super(key: key)` can be replaced by `(super.key)`)

Reply:
fixed